libs/domains/application/src/lib/slices/applications.slice.ts

```diff
@@ -183,9 +198,12 @@ export const fetchApplicationDeployments = createAsyncThunk<
   DeploymentHistoryApplication[],
   { applicationId: string; serviceType?: ServiceTypeEnum; silently?: boolean }
 >('application/deployments', async (data) => {
+  console.log(data.serviceType)
   let response
   if (data.serviceType === ServiceTypeEnum.CONTAINER) {
```

Comment:
For these 3 conditions can we have an util somewhere that contains these 3 methods:
isContainer()
isJob()
isApplication() ?

Like this it would be factorised somewhere and if one day the condition to detect the type changes, we just have to change it in one place. What do you think?